docs/BR.md

```diff
@@ -853,30 +853,26 @@ The CA SHALL update and reissue CRLs at least (i) once every twelve months and (
 No stipulation.
 
 ### 4.9.9 On-line revocation/status checking availability
-OCSP responses MUST conform to RFC6960 and/or RFC5019. OCSP responses MUST either:
-
-1. Be signed by the CA that issued the Certificates whose revocation status is being checked, or
-2. Be signed by an OCSP Responder whose Certificate is signed by the CA that issued the Certificate whose
-revocation status is being checked.
-
-In the latter case, the OCSP signing Certificate MUST contain an extension of type id-pkix-ocsp-nocheck, as
-defined by RFC6960.
+The CA SHALL provide an OCSP responder that conforms to the profile specified by RFC5019 for all Certificates issued, including the transport profile.
 
 ### 4.9.10 On-line revocation checking requirements
-Effective 1 January 2013, the CA SHALL support an OCSP capability using the GET method for Certificates issued
-in accordance with these Requirements.
+All OCSP responses MUST conform to the profile specified in Section 7.3
 
-For the status of Subscriber Certificates:
+For newly issued Subscriber Certificates, OCSP responses SHOULD be published prior to the delivery of the Certificate to the Subscriber. OCSP responses MUST be published no later than 10 minutes after the Subscriber Certificate has been issued.
```

Comment:
I feel like saying "directly" opens up rules-lawyering. Better to take the position that "delivering the Certificate to the Subscriber" should be **interpreted as** being made available through the interface used to request a certificate.

 - for ACME, that means GETting the certificate returns success rather than an in-progress code
 - for a web interface, a download link is provided
 - for issuance via email, the email with the certificate is sent to the requester